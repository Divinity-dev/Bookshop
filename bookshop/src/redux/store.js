import { configureStore } from '@reduxjs/toolkit';
import BookSlice from './Book/BookSlice';
import BookdetailSlice from './Book/BookdetailSlice';

const store = configureStore({
  reducer: {
    book: BookSlice,
    bookDetails: BookdetailSlice,
  },
});
export default store;
