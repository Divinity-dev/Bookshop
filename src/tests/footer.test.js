/* eslint-disable import/extensions */
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../components/footer';
import store from '../redux/store';

it('footer page renders correctly', () => {
  const tree = renderer
    .create(
      <React.StrictMode>
        <Provider store={store}>
          <BrowserRouter>
            <Footer />
          </BrowserRouter>
        </Provider>
      </React.StrictMode>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
