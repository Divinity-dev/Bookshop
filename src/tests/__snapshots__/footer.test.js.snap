// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer page renders correctly 1`] = `
<div
  className="footer"
>
  <h2>
    Bookshop 2023Â©
  </h2>
</div>
`;
